app/retail/views.py

```diff
@@ -1654,51 +1613,120 @@ def tribes(request):
-            'img': static('v2/images/tribes/landing/workshop.svg'),
+            'img': static('v2/images/workshops.svg'),
             'info': 'Host workshops and learn together'
         },
         {
             'title': 'Chat',
-            'img': static('v2/images/tribes/landing/chat.svg'),
+            'img': static('v2/images/Chat.svg'),
             'info': 'Direct connection to your trusted tribe'
         },
         {
             'title': 'Town Square',
-            'img': static('v2/images/tribes/landing/townsquare.svg'),
-            'info': 'Broadcast your priorities and engagey our tribe'
+            'img': static('v2/images/townsquare.svg'),
+            'info': 'Engage the members of your tribe'
         },
         {
             'title': 'Payout/Fund',
-            'img': static('v2/images/tribes/landing/payout.svg'),
+            'img': static('v2/images/payout.svg'),
             'info': 'Easily co-manage hackathons with your team'
         },
         {
             'title': 'Stats Report',
-            'img': static('v2/images/tribes/landing/stats.svg'),
+            'img': static('v2/images/report.svg'),
             'info': 'See how your hackathons are performing'
+        },
+        {
+            'title': 'Kudos',
+            'img': static('v2/images/kudos.svg'),
+            'info': 'Show appreciation to your tribe members'
+        }
+    ]
+
+    press = [
+        {
+            'link': 'https://twit.tv/shows/floss-weekly/episodes/474',
+            'img': 'v2/images/press/floss_weekly.jpg'
+        },
+        {
+            'link': 'https://epicenter.tv/episode/257/',
+            'img': 'v2/images/press/epicenter.jpg'
+        },
+        {
+            'link': 'http://www.ibtimes.com/how-web-30-will-protect-our-online-identity-2667000',
+            'img': 'v2/images/press/ibtimes.jpg'
+        },
+        {
+            'link': 'https://www.forbes.com/sites/jeffersonnunn/2019/01/21/bitcoin-autonomous-employment-workers-wanted/',
+            'img': 'v2/images/press/forbes.jpg'
+        },
+        {
+            'link': 'https://unhashed.com/cryptocurrency-news/gitcoin-introduces-collectible-kudos-rewards/',
+            'img': 'v2/images/press/unhashed.jpg'
+        },
+        {
+            'link': 'https://www.coindesk.com/meet-dapp-market-twist-open-source-winning-developers/',
+            'img': 'v2/images/press/coindesk.png'
+        },
+        {
+            'link': 'https://softwareengineeringdaily.com/2018/04/03/gitcoin-open-source-bounties-with-kevin-owocki/',
+            'img': 'v2/images/press/se_daily.png'
+        },
+        {
+            'link': 'https://www.ethnews.com/gitcoin-offers-bounties-for-ens-integration-into-dapps',
+            'img': 'v2/images/press/ethnews.jpg'
+        },
+        {
+            'link': 'https://www.hostingadvice.com/blog/grow-open-source-projects-with-gitcoin/',
+            'img': 'v2/images/press/hosting-advice.png'
         }
     ]
```

Comment:
^ This is duplicated information. Move it to separate file and import it. aka DRY code